src/cascadia/LocalTests_TerminalApp/CommandlineTest.cpp

```diff
@@ -0,0 +1,802 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include <WexTestClass.h>
+
+#include "../TerminalApp/AppCommandlineArgs.h"
+
+using namespace WEX::Logging;
+using namespace WEX::Common;
+using namespace WEX::TestExecution;
+
+using namespace winrt::TerminalApp;
+using namespace ::TerminalApp;
+
+namespace TerminalAppLocalTests
+{
+    // TODO:microsoft/terminal#3838:
+    // Unfortunately, these tests _WILL NOT_ work in our CI. We're waiting for
+    // an updated TAEF that will let us install framework packages when the test
+    // package is deployed. Until then, these tests won't deploy in CI.
+    class CommandlineTest

```

Comment:
We should probably make a fuzzer for this.

Reply:
-> #4169